### PR TITLE
[ISSUE #6116] fix the download links for and the sequence of source and sink

### PR DESCRIPTION
**Pull request by sijia-w**
Fix: #6116 

### Motivation
Fix the issue 6116.

### Modifications
Fix the duplicate links for source and link download and put them in alphabetical order.

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: site2/website/data/connectors.js [hunk 1]
```diff
@@ -13,15 +13,27 @@ module.exports = [
...
-        longName: 'Apache Cassandra sink',
-        type: 'Sink',
+        longName: 'Apache Cassandra source and sink',
+        type: 'Source and sink',
         link: 'https://cassandra.apache.org'
     },
     {
-        name: 'data-generator',
-        longName: 'Test Data Generator source',
+        name: 'kafka-connect-adaptor',
```
Comment (vzhikserg) [inline site2/website/data/connectors.js:21]: The connectors were in alphabetical order. Was it changed intentionally?

File: site2/website/data/connectors.js [hunk 2]
```diff
@@ -13,15 +13,27 @@ module.exports = [
     },
     {
         name: 'cassandra',
-        longName: 'Apache Cassandra sink',
-        type: 'Sink',
+        longName: 'Apache Cassandra source and sink',
+        type: 'Source and sink',
```
Comment (vzhikserg) [inline site2/website/data/connectors.js:17]: I am really curious if this field is used anywhere.

Comment (sijia-w) [reply site2/website/data/connectors.js:17]: I have checked it locally. The links all work well.

Comment (vzhikserg) [reply site2/website/data/connectors.js:17]: This comment was not really related to  your changes. The _type_ field is not used and maybe it makes sense to remove it in the future

Comment (sijia-w) [reply site2/website/data/connectors.js:17]: Okay. Thanks so much for your reply.

**Pull request merged**